gptel-context.el

```diff
@@ -91,8 +91,10 @@ context chunk.  This is accessible as, for example:
   context.  If there is already a gptel context at point, remove it
   instead.
 
-- If in Dired, add marked files or file at point to the context.
-  With negative prefix ARG, remove them from the context instead.
+- If in Dired, add marked files or file at point to the context. If
+  the file at point is a directory, or directories are marked,
```

Comment:
This is difficult to read.  How about

```
- If in Dired, add marked files or file at point to the context.  Directories
  will have all their files added recursively.  With negative prefix ARG, remove
  them from the context instead.
```

Reply:
Looks good.

Reply:
Addressed: [3e5c1e6](https://github.com/karthink/gptel/pull/438/commits/8e5e3c867131f0c795c7eb62dd8ba28b25a9cf27)